### PR TITLE
Added documentation for return type checker

### PR DESCRIPTION
**Pull request by malukenho**

### CODE REVIEW

**Comment by malukenho** [comment]
linking #37

File: README.md [hunk 1]
```diff
@@ -90,6 +90,40 @@ $object->integer = '123';
 Please note that this kind of feature currently only works with public and 
 protected properties.
 
+#### Return type checks
+
+Quite similar to above functionality, this feature will prevent your application
```
Comment (Ocramius) [inline README.md]: the above functionality

File: README.md [hunk 2]
```diff
@@ -90,6 +90,40 @@ $object->integer = '123';
 Please note that this kind of feature currently only works with public and 
 protected properties.
 
+#### Return type checks
+
+Quite similar to above functionality, this feature will prevent your application 
+from **return** illegal values from methods that are type-hinted (via docblock)
```
Comment (Ocramius) [inline README.md]: returning

File: README.md [hunk 3]
```diff
@@ -90,6 +90,40 @@ $object->integer = '123';
...
+(new Example())->dummyReturn('string');
+```
+
+Following code will crash:
+
+```php
+(new Example())->dummyReturn(123);
+```
+
+Please note that this kind of feature currently only works with public and
```
Comment (Ocramius) [inline README.md]: Mproperties or methods?

**Pull request merged**